pkg/descheduler/strategies/nodeutilization/lownodeutilization_test.go

```diff
@@ -48,14 +48,16 @@ func TestLowNodeUtilization(t *testing.T) {
 
 	testCases := []struct {
 		name                         string
+		useDeviationThresholds       bool
 		thresholds, targetThresholds api.ResourceThresholds
 		nodes                        []*v1.Node
 		pods                         []*v1.Pod
 		expectedPodsEvicted          uint
 		evictedPods                  []string
 	}{
 		{
-			name: "no evictable pods",
+			name:                   "no evictable pods",
+			useDeviationThresholds: false,
```

Comment:
`false` is the default value, no need to set `useDeviationThresholds` explicitly. The same for other occurrences.